src/styles/components/_card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-bottom: 0;
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    .uppercase {
+      text-transform: uppercase;
+    }
+  }
+
+  &__product-code {
+    margin-bottom: 0;
+    margin-top: 4px;
+
+    font-weight: 400;
+    line-height: 14px;
+    color: getColor("secondary-text");
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-coun {
+      margin: 0;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 0;
+    margin-top: 24px;
+
+    &-title,
```

Comment:
and delete it

Reply:
I can't delete it? becouse when I add p starts has a margin top and bottom by defolt, so this is need